File: python/Iniciante/URI1035.py
linha = input().split()
a, b, c, d = list(map(int, linha))

if b > c and d > a and (c + d) > (a + b) and c > 0 and d > 0 and a % 2 == 0:
  print("Valores aceitos")
else:
  print("Valores nao aceitos")

File: java/iniciante/URI1035.java
package URI;

import java.util.Scanner;

public class URI1035 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int A = scanner.nextInt(), B = scanner.nextInt(), C = scanner.nextInt(), D = scanner.nextInt();

        if ((B > C) && (D > A) && (C + D > A + B) && (C > 0) && (D > 0) && (A % 2 == 0)) {
            System.out.println("Valores aceitos");
        } else {
            System.out.println("Valores nao aceitos");
        }
    }
}
